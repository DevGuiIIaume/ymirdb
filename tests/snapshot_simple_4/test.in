set a 1
set b 2
set c 3
set d 4
set e 5
set f 6
list entries
del a
del f
list entries
del c
del d
list entries
del e
del b
set g 7
list entries
bye
