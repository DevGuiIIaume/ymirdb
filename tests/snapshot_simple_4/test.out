> ok

> ok

> ok

> ok

> ok

> ok

> f [6]
e [5]
d [4]
c [3]
b [2]
a [1]

> ok

> ok

> e [5]
d [4]
c [3]
b [2]

> ok

> ok

> e [5]
b [2]

> ok

> ok

> ok

> g [7]

> bye
